package/kernel/linux/modules/crypto.mk

```diff
@@ -572,7 +572,7 @@ endef
 ifndef CONFIG_TARGET_uml
 define KernelPackage/crypto-lib-chacha20/x86_64
   KCONFIG+=CONFIG_CRYPTO_CHACHA20_X86_64
-  FILES+=$(LINUX_DIR)/arch/x86/crypto/chacha-x86_64.ko
+  FILES+=$(LINUX_DIR)/arch/x86/crypto/chacha-x86_64.ko@lt6.18
```

Comment:
It makes more sense to specify `DEPENDS:=@!LINUX_6_18` rather than excluding ko file (no file left). 
Similarly in some changes below.